smartsim/settings/alpsSettings.py

```diff
@@ -42,10 +42,19 @@ def __init__(self, exe, exe_args=None, run_args=None, env_vars=None):
         :type run_args: dict[str, str], optional
         :param env_vars: environment vars to launch job with, defaults to None
         :type env_vars: dict[str, str], optional
+        :param block_in_batch: whether the execution of this call should block when executed
+                               in a batch.
+        :type block_in_batch: bool
+        :param expand_exe: whether the executable path should be expanded. It is recommended
```

Comment:
Would the alternative be that it's in your ``PATH``?

Reply:
Actually the problem with this is that if it is an executable you have in another conda (or virtual) env, you cannot expand it here.